### PR TITLE
build: four fedimintds for dev env

### PR DESCRIPTION
**Pull request by EthnTuttle**
I used four fedimints since I imagine this being the "standard". We can now also dev on UI/UX for when consensus concerns arise by turning off a fedimint or two.

Marking Draft because I haven't actually tested a setup with this, but wanted to get it out there.

### CODE REVIEW

**Comment by EthnTuttle** [comment]
Tested. Works.

File: mprocs.yml [hunk 1]
```diff
@@ -19,6 +19,18 @@ procs:
       PORT: '3001'
       REACT_APP_FM_CONFIG_API: ws://127.0.0.1:18184
       BROWSER: none
+  guardian-ui-3:
+    shell: yarn dev:guardian-ui
+    env:
+      PORT: '3003'
```
Comment (okjodom) [inline mprocs.yml:25]: nit: maybe we need to assign contiguous ports to the guardian UIs thus push gateway UI port back to 3004 or 3005?

**Review by okjodom — DISMISSED** [review]
validated. left a nit/suggestion about gateway ui port

**Comment by okjodom** [comment]
nit: should also add all `fm_*` dirs to gitignore

File: mprocs.yml (hunk 1 above)
Comment (EthnTuttle) [reply mprocs.yml]: Good call. Was lazy. 😅

**Comment by EthnTuttle** [comment]
@okjodom Addressed the nits. Good call so we can keep this repo well "gardened".

**Review by okjodom — APPROVED** [review]

**Pull request merged**